### ISSUE TITLE
Design System Update: Update background colors shown on new homepage hero veteran images

### ISSUE THREAD

**Issue by thejordanwood** [department-of-veterans-affairs/va.gov-team#52365]
To learn more about contributing to the Design System or requesting new components, requesting changes to existing components, and reporting bugs, go here.
https://design.va.gov/about/contributing-to-the-design-system/

---
# Design System update

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## Proposed Changes

On the new homepage: when no V2 Home page benefit promo custom block is published to the Home page benefit promo entity queue, a series of veteran images will show:
https://www.sketch.com/s/3aa40506-4be2-46cc-876b-93f1a9f3a857/a/WKW7xVA

During prototype build, the background colors for images in that space were hard coded. Those colors are not in the design system and are unique to this homepage design. These colors should be moved into the design system. 

<img width="650" alt="Screenshot 2023-01-20 at 1 07 29 PM" src="https://user-images.githubusercontent.com/104526584/213785027-d5c4e2de-e92d-488b-98e0-543ecbad5a01.png">

**Comment by thejordanwood** [comment]
I believe @rtwell originally created this design and @jilladams has been doing work for the new homepage too. Tagging them here to keep them in the loop.

**Comment by thejordanwood** [comment]
I believe @rtwell originally created this design and @jilladams has been doing work for the new homepage too. Tagging them here to keep them in the loop.

(Accidentally closed this issue, but am reopening it now)

**Comment by humancompanion-usds** [comment]
@thejordanwood - What are the hex values for the colors being requested and what are their token names from the USWDS? Thanks.

**Comment by thejordanwood** [comment]
@humancompanion-usds The 3 hex values being requested are #084778, #0C3D69, and #0F345A. These are the colors that appear behind the images of the Veterans in the homepage hero. I don't see token names that correspond to these colors on the [USWDS design token list](https://designsystem.digital.gov/design-tokens/color/system-tokens/#using-color-tokens), but I can alter the hex values slightly to get them to align with a token if needed.

**Comment by humancompanion-usds** [comment]
@thejordanwood - Yes, colors need to come from our system or the USWDS. Please adjust as needed and let me know where you land and then we can add those color tokens. Thanks.

**Comment by thejordanwood** [comment]
@humancompanion-usds After looking at this more closely, I think we can use some colors that are already in the system. For the lighter blue, we can use #0071BB (color-primary) and for the darker blue we can use #003e73 (color-primary-darker) from our design system. 

The middle blue can change to #005EA2 (blue-60v). I was able to access that blue in the Sketch file design system but can't find it in the [VA color palette](https://design.va.gov/foundation/color-palette). Is this something we already have or will that need to be added?

<img width="721" alt="Screenshot 2023-01-23 at 12 53 05 PM" src="https://user-images.githubusercontent.com/104526584/214125891-b017ee5e-5eb4-46c7-9b12-4c23ea40f351.png">

**Comment by thejordanwood** [comment]
Thanks for the suggestion @jhonnyoddball. This is what that color looks like for the middle piece. It looks a little out of place because it's not the same tone of blue. I was looking at colors in the vivid blue palette to try and find something close. 

<img width="725" alt="Screenshot 2023-01-23 at 3 28 18 PM" src="https://user-images.githubusercontent.com/104526584/214160730-147ff38d-d8a8-4e56-a853-fa4a219cb02c.png">

**Comment by jhonnyoddball** [comment]
@thejordanwood Do still need a new color for that?

**Comment by thejordanwood** [comment]
@jhonnyoddball Yes please! Can we add the color #005EA2 (blue-60v)?

**Comment by jhonnyoddball** [comment]
@thejordanwood That color does not exist in `formation`. Is it possible to hard-code that color since your team is going to be the only ones using it?

We are in the process to replace `formation` with a `css-library` in which will have all the version 3 colors from USWDS.

**Comment by jilladams** [comment]
Yes, these colors are currently hard coded for us, and just seeking to get them into the design system so we're not operating outside the process. If you're content for us to be an exception, that works for us, for whichever of these colors that's necessary/best.

**Comment by jhonnyoddball** [comment]
> Yes, these colors are currently hard coded for us, and just seeking to get them into the design system so we're not operating outside the process. If you're content for us to be an exception, that works for us, for whichever of these colors that's necessary/best.

We are more concerned that you/we are using USWDS colors. We will be adding those colors but it's in the roadmap right now. You can go ahead and use the 2 tokens that already exist and hard-code the one that doesn't exist.

**Comment by caw310** [comment]
Closing this issue as instructions have been provided to accommodate this request. 

DST will upgrade to use USWDS v3 colors in the future.
